### ISSUE TITLE
Training not converging

### ISSUE THREAD

**Issue by pchanda** [fenghansen/ESRGAN-Keras#5]
The training code 'RTC-SR.py' does not work as the required weights 'SA-RRDB.h5' is not provided in data/weights in the repo.

I tried using './data/weights/DIV2K_gan.h5' instead, training did not converge, I am getting the following after ~10K iterations (see below). I am using DIV2K+my own images for training. 

Any help highly appreciated. 

....
Epoch 10000/100000 | Time: 102s
>> Generator/GAN: loss=0.1328, Adversarial_loss=9.0670, Content_loss=17.5009
>> Discriminator: loss=0.0436, acc=0.9888
--
Epoch 10025/100000 | Time: 121s
>> Generator/GAN: loss=0.1176, Adversarial_loss=6.9390, Content_loss=16.5896
>> Discriminator: loss=0.0138, acc=0.9971
--
Epoch 10050/100000 | Time: 101s
>> Generator/GAN: loss=0.1161, Adversarial_loss=6.2183, Content_loss=17.0035
>> Discriminator: loss=0.0318, acc=0.9918
--
Epoch 10075/100000 | Time: 118s
>> Generator/GAN: loss=0.1249, Adversarial_loss=7.6172, Content_loss=17.3540
>> Discriminator: loss=0.0023, acc=0.9994

But I do not find the RRDB weights, that you mentioned you uploaded. The folder data/weights only has DIV_2k weights.
